packages/android_intent/android/src/main/java/io/flutter/plugins/android_intent/AndroidIntentPlugin.java

```diff
@@ -0,0 +1,85 @@
+package io.flutter.plugins.android_intent;
```

Comment:
Seems dart src files do not need this?

Also, can we make it part of flutter create?